ydb/mvp/meta/mvp.cpp

```diff
@@ -208,6 +211,13 @@ void TMVP::TryGetGenericOptionsFromConfig(
             TYdbLocation::UserToken = auth["token"].as<std::string>("");
```

Comment:
I think this part becomes obsolete and should be replaced with the name of default token? or should be removed completely

Reply:
removed